httprest/pkg/user/service.go

```diff
@@ -0,0 +1,52 @@
+package user
+
+import (
+	"context"
+
+	gokitlog "github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+type Service interface {
+	CreateUser(ctx context.Context, request CreateUserRequest) (CreateUserResponse, error)
+}
+
+type service struct {
+	repository Service
+	logger     gokitlog.Logger
+}
+
+type CreateUserRequest struct {
```

Comment:
Maybe you could have only one entity User and use it through the layers avoiding repeating the creation of this structures.